### PR TITLE
HHH-12313: org.hibernate.jpa.test.transaction.TransactionCommitFailureTest fails on HANA

### PR DESCRIPTION
**Pull request by breglerj**

### CODE REVIEW

**Comment by vladmihalcea** [comment]
Applied upstream, thanks

File: hibernate-core/src/main/java/org/hibernate/dialect/AbstractHANADialect.java [hunk 1]
```diff
@@ -1497,8 +1501,11 @@ public void contributeTypes(TypeContributions typeContributions, ServiceRegistry
 					}
 				}
 			}
-			catch (SQLException e) {
-				// ignore
+			catch (Exception e) {
+				LOG.debug(
+						"An error occurred while trying to determing the value of the HANA parameter indexserver.ini / session / max_lob_prefetch_size. Using the default value "
```
Comment (gsmet) [inline hibernate-core/src/main/java/org/hibernate/dialect/AbstractHANADialect.java]: @vladmihalcea just noticed a typo here: s/determing/determine/

Comment (gsmet) [reply hibernate-core/src/main/java/org/hibernate/dialect/AbstractHANADialect.java]: Yeah, don't bother, I'd better push a fix myself :). Too used to the NoORM process :]

Comment (gsmet) [reply hibernate-core/src/main/java/org/hibernate/dialect/AbstractHANADialect.java]: Just pushed the fix.

Comment (breglerj) [reply hibernate-core/src/main/java/org/hibernate/dialect/AbstractHANADialect.java]: ok :-)

**Pull request closed without merging**